doc/documentation_data_format.md

```diff
@@ -314,24 +332,27 @@ Additional columns may be added.
   a possible optimization and for the objective function. Priors which are 
   only used for sampling of initial starting points or only for optimization
   should be specified in the additional columns `initializationPriorType` or
-  `objectivePriorType`, respectivly. Possible prior types are (see Extensions):
-  
-    - uniform: flat prior on linear parameters
-    - normal: Gaussian prior on linear parameters
-    - laplace: Laplace prior on linear parameters
-    - logNormal: exponentiated Gaussian prior on linear parameters
-    - logLaplace: exponentiated Laplace prior on linear parameters
-    - parameterScaleUniform (default): Flat prior on original parameter 
-    scale (equivalent to "no prior")
-    - parameterScaleNormal: Gaussian prior on original parameter scale
-    - parameterScaleLaplace: Laplace prior on original parameter scale
+  `objectivePriorType`, respectively.
+
+  Possible prior types are (see also Extensions):
+
+    - *uniform*: flat prior on linear parameters
+    - *normal*: Gaussian prior on linear parameters
+    - *laplace*: Laplace prior on linear parameters
+    - *logNormal*: exponentiated Gaussian prior on linear parameters
+    - *logLaplace*: exponentiated Laplace prior on linear parameters
```

Comment:
why don't we use snake_case here?

Reply:
Because we don't use it anywhere in the column headers. Meanwhile I wish we had used it there :). But the initial capital is inconsistent, agreed. Maybe to be be discussed elsewhere?